app/views/users/_form.html.erb

```diff
@@ -0,0 +1,10 @@
+<% with_options scope: 'contribute.pages.users.new.form' do |i18n| %>
+  <%= simple_form_for @user, url: { action: @user.persisted? ? :update : :create } do |user| %>
+    <%= user.input(:email, label: i18n.t('labels.email')) %>
```

Comment:
Can't we rely on SimpleForm's i18n key deduction here? e.g. by moving the labels to `contribute.form.labels.user`

Reply:
I just added to keys to localeapp, but I'm not sure how these are supposed to be wired up without adding the scope as above.

Reply:
With the following changes, the labels will all be picked up automatically from  the `contribute.form.labels.user` scope.

*Create app/helpers/simple_form_helper.rb:*
```ruby
# frozen_string_literal: true

module SimpleFormHelper
  # Override +SimpleForm::ActionViewExtensions::FormHelper::simple_form_for+ to
  # use +ApplicationFormBuilder+ as the default form builder.
  def simple_form_for(record, options = {}, &block)
    options[:builder] ||= ApplicationFormBuilder
    super(record, options, &block)
  end
end
```

*Update app/builders/application_form_builder.rb:*
```ruby
# frozen_string_literal: true

class ApplicationFormBuilder < SimpleForm::FormBuilder
  def find_input(*_)
    super.tap do |input|
      def input.i18n_scope
        'contribute.form'
      end

      # Override +SimpleForm::Inputs::Base#translate_from_namespace+ to log
      # lookup candidates to facilitate development.
      #
      # Toggled on by setting ENV['ENABLE_SIMPLE_FORM_I18N_LOGGING']
      if ENV['ENABLE_SIMPLE_FORM_I18N_LOGGING']
        def input.translate_from_namespace(namespace, default = '')
          model_names = lookup_model_names.dup
          lookups     = []

          until model_names.empty?
            joined_model_names = model_names.join('.')
            model_names.shift

            lookups << :"#{joined_model_names}.#{lookup_action}.#{reflection_or_attribute_name}"
            lookups << :"#{joined_model_names}.#{reflection_or_attribute_name}"
          end
          lookups << :"defaults.#{lookup_action}.#{reflection_or_attribute_name}"
          lookups << :"defaults.#{reflection_or_attribute_name}"
          lookups << default

          # begin custom logging code
          log_msg = "#{object.class.inspect}.#{attribute_name} SimpleForm i18n:\n" \
                    "* scope: #{i18n_scope}.#{namespace}\n" \
                    "* lookups:\n" +
                    lookups.reject(&:blank?).map { |l| "  - #{l}" }.join("\n")
          Rails.logger.debug(log_msg)
          # end custom logging code

          I18n.t(lookups.shift, scope: :"#{i18n_scope}.#{namespace}", default: lookups).presence
        end
      end
    end
  end
end
```